HW06.Rmd

```diff
@@ -142,38 +149,41 @@ Now create a graphing function to create boxplots to visualize the results. Plot
 ```{r create a graphing function}
 #to work in the tidyverse, it will be easier to make tidy the dataframe first
 
+#For this last part, I flipped the entire GBM.genes.of.interest data frame and the I create a new parameter that considers if the data came from the WT or the H3K27 data frame, which is going to be useful to add a color aesthetic the the plots.
 GBM.tidied <- GBM.genes.of.interest %>%
   gather(key = "file", value = "value", 2:46) %>%
   separate(file, sep = "/", into = c("file", "H3K27")) %>%
   spread(key = gene_symbol, value = value)
 
-GBM.tidied$H3K27[GBM.tidied$H3K27 == "H3"] <- "H3K27_Mutated"
+GBM.tidied$H3K27[GBM.tidied$H3K27 == "H3"] <- "H3K27_Mutated" #In order to have the labels as requested
 
-theme_set(theme_bw())
+theme_set(theme_bw()) #General theme of the plot
 
 
 #create a graphing function
 
 graph_function <- function(gene = "name") {
-  ggplot(GBM.tidied, aes_string(x = "H3K27", y = gene, fill = "H3K27"))+

```

Comment:
aes_string() is such a useful looking function. Definitely need to look into that because this is a very clean and short solution